science-platform/values-usdfdev.yaml

```diff
@@ -0,0 +1,70 @@
+environment: usdfdev
+fqdn: usdf-rsp-dev.slac.stanford.edu
+vault_path_prefix: secret/rubin/usdf-rsp-dev
+# butlerRepositoryIndex: "s3://butler-us-central1-repo-locations/data-repos.yaml"
+repoURL: https://github.com/slaclab/phalanx.git
+
+alert_stream_broker:
+  enabled: false
+cachemachine:
+  enabled: true
+cert_manager:
+  enabled: false
+datalinker:
+  enabled: true
+exposurelog:
+  enabled: false
+gafaelfawr:
+  enabled: true
+hips:
+  enabled: false
+ingress_nginx:
+  enabled: false
```

Comment:
ingress-nginx has to be enabled. We don't support an environment in which we're not managing the ingress. Gafaelfawr is not supported in that configuration.